package.json

```diff
@@ -36,11 +36,12 @@
     "lodash.isnil": "4.0.0",
     "plotly.js-dist": "2.17.0",
     "prop-types": "15.7.2",
-    "react": "16.14.0",
-    "react-dom": "16.14.0",
+    "react": "17.0.2",
+    "react-dom": "17.0.2",
     "react-fullstory": "1.4.0",
-    "react-helmet": "5.2.1",
+    "react-helmet": "6.1.0",
     "react-instantsearch-dom": "6.38.1",
+    "react-lines-ellipsis": "0.15.3",
```

Comment:
Would recommend trying to use Paragon's `Truncate` component, if possible, rather than relying on a new third-party library.

Reply:
Sure that's better, I have replaced it with Paragon's `Truncate`